scripts/insert_snapshot_warning.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const fs = require('fs');
+const { JSDOM } = require('jsdom');
+const { execSync } = require('child_process');
+
+const COMMIT = execSync('git rev-parse --verify HEAD').toString();
```

Comment:
```suggestion
const COMMIT = String(execSync('git rev-parse --verify HEAD'));
```

Reply:
any particular reason for this?

Reply:
oh this is just a nit; mainly that calling .toString doesn’t ensure you get a string out; only String() or interpolating into a template literal both ensures you call toString and coerce to a string - so i have a general code review policy of “never call toString with no args directly”.